src/ApiConnectors/BrowseDataApiConnector.php

```diff
@@ -69,7 +69,9 @@ public function getBrowseData(string $code, array $columns, array $sortFields =
     {
         Assert::minCount($columns, 1);
         Assert::allIsInstanceOf($columns, BrowseColumn::class);
-        Assert::allIsInstanceOf($sortFields, BrowseSortField::class);
+        if(count($sortFields)) {
```

Comment:
This isn't really needed right - an empty collection of $sortFields are also all of the `BrowseSortField` class.